broadinstitute/single_cell_portal_core
# consolidating scatter plot tests, adding cache test (SCP-3897)

This adds tests for two of the bugs we found prior to yesterdays release-- the scatter color reversal, and the caching of user annotations. 

It also consolidates the scatter plot tests and test data into the test/js/visualization directory

TO TEST:
 1. run `yarn test scatter-plot`
 2. observe passing
 3. run `yarn test plot-data-cache`
 4. observe passing
